wpxmlrpc.podspec

```diff
@@ -1,16 +1,15 @@
 Pod::Spec.new do |s|
   s.name         = "wpxmlrpc"
-  s.version      = "0.8"
+  s.version      = "0.8.2"
   s.summary      = "Lightweight XML-RPC library."
   s.homepage     = "https://github.com/wordpress-mobile/wpxmlrpc"
-  s.license      = { :type => 'MIT', :file => 'LICENSE.md' }
-  s.author       = "WordPress"
+  s.license      = { :type => 'GPLv2', :file => 'LICENSE.md' }
+  s.author       = {"WordPress", "Automattic" => "mobile@automattic.com"}
   s.source       = { :git => "https://github.com/wordpress-mobile/wpxmlrpc.git", :tag => "#{s.version}" }
   s.source_files = 'WPXMLRPC'
-  s.public_header_files = [ 'WPXMLRPC/WPXMLRPC.h', 'WPXMLRPC/WPXMLRPCEncoder.h', 'WPXMLRPC/WPXMLRPCDecoder.h' ]
```

Comment:
This exposes a few header files that aren't documented, and even if they where they just make the API more confusing since they're not relevant to users of this library.

This brings up an interesting discussion about testing, and I was talking with @astralbodies about this yesterday. 

**Should we only test our public interfaces?**

If this is the case, we'd need to refactor our tests so they only use the public headers. So, instead of calling `WPXMLRPCDataCleaner` directly, we'd test `WPXMLRPCDecoder` to see if it can handle _dirty_ data.

If we consider that it's simpler/better to test these internal classes separately, then the test project should test the sources directly and not the pod.

In no case we should expose internal classes to the public _just_ to make testing more convenient.